Agrume/Agrume.swift

```diff
@@ -217,6 +220,26 @@ public final class Agrume: UIViewController {
     }
     return _spinner!
   }
+	
+	fileprivate var _closeBtn: UIButton?
+	fileprivate var closeBtn: UIButton {
+		if _closeBtn == nil {
+			let closeBtn = UIButton(type: .system)
+			closeBtn.setImage(UIImage(named: "close_btn"), for: UIControlState.normal)
```

Comment:
If I remember correctly, there's an issue with asset catalogs in CocoaPods. You need to load the image using `UIImage(named: "", in: …)` and add it to the resources definition of the podspec.

This raises the issue that I have with this in general: I'd much prefer this being something the user of the library can pass in, rather than having a certain visual style forced upon them. That would also allow using a text button instead.